module/module.go

```diff
@@ -153,13 +153,12 @@ type peerResourceState struct {
 	// NOTE As I'm only suppporting video to start this will always be a single element
 	// once we add audio we will need to make this a slice / map
 	subID rtppassthrough.SubscriptionID
-	// NOTE As I'm only suppporting video to start this will always be a single element
-	// once we add audio we will need to make this a slice
-	sender *webrtc.RTPSender
 }
 
 // Module represents an external resource module that services components/services.
 type Module struct {
+	ctx                     context.Context
```

Comment:
We need to know when the module is shutting down so we can close all the rtppassthrough subscriptions.